datacats/docker.py

```diff
@@ -103,14 +103,15 @@ def _get_docker():
 
 class WebCommandError(Exception):
 
-    def __init__(self, command, container_id, logs):
+    def __init__(self, command, container_id, logs, ssh=False):
```

Comment:
Yes, I couldn't find out a way to differentiate them though

That's what I need feedback on :)